blocks/core/ff_module/ff_module-button/ff_module-button.rt

```diff
@@ -1,15 +1,18 @@
-<button type="button"
+<button
+    type="button"
     title="{this.props.text}"
     id="{this.props.id}"
     disabled="{this.props.disabled}"
-    class="ff_module-button{this.props.modifier ? ' ff_module-button--'+this.props.modifier : ''}{this.props.classes ? ' '+this.props.classes : ''}{this.props.disabled ? ' ff_module-button--is-disabled' : ''}"
+    class="{this.generateClass('ff_module-button', this.props)}"
```

Comment:
Not strictly, no, for the props. I've kept the base separate just for consistency as that's a common pattern in the components and often it does change within a component to apply to different elements.